numba/cuda/cudadrv/devicearray.py

```diff
@@ -410,6 +432,86 @@ def _numba_type_(self):
         """
         return numpy_support.from_dtype(self.dtype)
 
+    def getitem(self, item, stream=0):
```

Comment:
I think this is just overriding `getitem` from `DeviceNDArrayBase` with an identical implementation. Can it be removed, or did I miss something here?